### DISCUSSION TITLE
KataGo for Life-and-Death Go

### DISCUSSION THREAD

**Discussion by hzyhhzy** [hzyhhzy/KataGo#20] [Announcements]
Life-and-Death Go is a variant of Go.    
Black has 17~20 initial stones. White's winning condition is having one alive part.    
![image](https://github.com/user-attachments/assets/8289b835-ed95-4698-9e6d-06e690b3692d)

   
    
This model is based on official KataGo's b28c512nbt using transfer-learning.    
   
In order to achieve a high level of Go strength at a very low cost, the early stages of training were done on a small 9x9 board, followed by 13x13 board, and finally 19x19 board.      
Additionally, the official KataGo weights were used for transfer learning on the "life-and-death" game rules, which significantly improved the model's strength.       
~ 1 week on 2x RTX4090    

<hr /><em>This discussion was created from the release <a href='https://github.com/hzyhhzy/KataGo/releases/tag/LifeGo_20241025'>KataGo for Life-and-Death Go</a>.</em>

**Comment by awsjgy** [comment]
死活对局trt版本用不了，opencl可以.，报错如下
![Uploading Screenshot_2024-10-26-03-09-07-42_2665fb67b16260a8d818298cef8dc107.jpg…]()
